src/connections.jl

```diff
@@ -16,20 +16,17 @@ function _connection_parameter_dict(;
 )
     keep_option((k, v)) = !(k == "TimeZone" && v == "")
 
-    Dict{String, String}(
+    return Dict{String, String}(
         "client_encoding" => client_encoding,
         "application_name" => application_name,
-        "options" => join(
-            imap(Iterators.filter(keep_option, connection_options)) do (k, v)
-                "-c $k=$(show_option(v))"
-            end,
-            " ",
-        ),
+        "options" => join(imap(Iterators.filter(keep_option, connection_options)) do (k, v)
+            "-c $k=$(show_option(v))"
+        end, " "),
```

Comment:
I think this is less clear than the previous formatting.

Reply:
i agree. Can you open a BlueStyle issue about this. If we can formulate guidance here, we can then implement in the Formatter

Reply:
What kind of issue? There are many interactions here. I think also that this is wrong according to existing style rules when applied to the `join` function.

Reply:
i think this is a consequence of the existing guidance. i.e. it's formatted to something that no BlueStyle guidance recommends against. (i still agree it's less clear than before... automatic style rules are difficult)

Reply:
oh yes, i don't understand why the Formatter put `),` on this same line...

Reply:
No the `join` call breaks this rule:

> Function calls which cannot fit on a single line within the line limit should be broken up such that the lines containing the opening and closing brackets are indented to the same level while the parameters of the function are indented one level further. In most cases the arguments and/or keywords should each be placed on separate lines.

Reply:
Can you open a JuliaFormatter issue with this snippet please?

Reply:
https://github.com/domluna/JuliaFormatter.jl/issues/283#issuecomment-698905099